utils/openshift_acme.py

```diff
@@ -1,76 +1,31 @@
 ACME_DEPLOYMENT = """
-apiVersion: extensions/v1beta1
 kind: Deployment
+apiVersion: apps/v1
 metadata:
   name: %(deployment_name)s
   labels:
     app: openshift-acme
 spec:
-  progressDeadlineSeconds: 600
-  replicas: 1
-  revisionHistoryLimit: 10
+  replicas: 2
   selector:
     matchLabels:
       app: openshift-acme
   strategy:
-    type: Recreate
+    type: RollingUpdate
   template:
     metadata:
-      creationTimestamp: null
       labels:
         app: openshift-acme
     spec:
       containers:
-      - env:
-        - name: OPENSHIFT_ACME_EXPOSER_IP
-          valueFrom:
-            fieldRef:
-              apiVersion: v1
-              fieldPath: status.podIP
-        - name: OPENSHIFT_ACME_ACMEURL
-          value: 'https://acme-v01.api.letsencrypt.org/directory'
-        - name: OPENSHIFT_ACME_LOGLEVEL
-          value: '4'
-        - name: OPENSHIFT_ACME_NAMESPACE
-          valueFrom:
-            fieldRef:
-              apiVersion: v1
-              fieldPath: metadata.namespace
-        image: %(image)s
+      - image: %(image)s
         imagePullPolicy: Always
         name: openshift-acme
-        ports:
-        - containerPort: 5000
-          protocol: TCP
-        resources:
```

Comment:
I think it'd be good to retain the resource limits for good measure

Reply:
done